virttest/utils_libvirt/utils_libvirt_nwfilter.py

```diff
@@ -0,0 +1,32 @@
+"""
+Accommodate libvirt nwfilter utility functions.
+
+:copyright: 2020 Red Hat Inc.
+"""
+
+import logging
+
+from virttest import virsh
+
+
+def clean_up_nwfilter_binding():
```

Comment:
consider providing the option for ignore_status to user via param here and default it it False and return the processed result.

Reply:
add one parameter ignore_status,but default value is True to allow silent failure. 
The reason behind the scene is that it is just clean up functionality, better not stop or fail case  execution at the beginning due to clean up failures

Reply:
it is better to notify the user and fail this testcase, rather leaving the unstable environment and subsequent test may result in failure or unknown behaviour.

If we expect the test case to clean the env it created it should do so, otherwise we should call it as error/failure based on the test steps.

Reply:
I understand there are two scenarios it may match:
1)Clean up dirty environment left by last case as pre-caution step, and I assume silent failure is reasonable
2)Clean up  environment created by this case itself, it should fail /error this 
So as you suggest, it provide ignore_status parameter to allow user to have flexisbility